source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,85 @@
+.. _csharp-update-many:
+
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can update more than one document using the `UpdateMany() <{+api-root+}/M_MongoDB_Driver_IMongoCollection_1_UpdateMany.htm>`__ method on
+a collection object.
```

Comment:
In a separate review, Nathan suggested changing ``MongoCollection`` to just "collection"